recipe/meta.yaml

```diff
@@ -181,6 +180,157 @@ outputs:
         Compiler for CUDA applications.
       doc_url: https://docs.nvidia.com/cuda/index.html
 
+  - name: cuda-crt-tools
+    files:
+      - bin/crt          # [linux]
+      - Library\bin\crt  # [win]
+    requirements:
+      host:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - cuda-version {{ cuda_version }}
+      run:
```

Comment:
We need a run dependency on `cuda-crt-dev` to meet the header dependency requirements

Reply:
As context to the reader Robert is commenting on `cuda-crt-tools`, which is just above the fold in the diff

So far the `*-tools` packages have just contained `bin` contents

For example `cuda-nvcc-tools`:

https://github.com/conda-forge/cuda-nvcc-impl-feedstock/blob/2e51f9306c8482fdc7b38c4caadf06b4f94aef7a/recipe/meta.yaml#L35-L38

More generally the packages in `cuda-nvcc-impl` are not necessarily intended for publicly consumption themselves. Instead they are intended to [construct packages in `cuda-nvcc`]( https://github.com/conda-forge/cuda-nvcc-feedstock/blob/fedbe034f3f53500f86ab7bad667e597ef3e4fe2/recipe/meta.yaml#L19 )

For example [`cuda-nvcc-tools` is used to construct `cuda-nvcc_{{ target_platform }}`]( https://github.com/conda-forge/cuda-nvcc-feedstock/blob/fedbe034f3f53500f86ab7bad667e597ef3e4fe2/recipe/meta.yaml#L60 ), which is publicly consumable (in CUDA 12 recipe builds `{{ compiler("cuda") }}` becomes `cuda-nvcc_{{ target_platform }}` )

IOW it is ok if this package is not usable itself. It's more important that we are putting it together with the right other pieces to construct usable packages from the users perspective

When it comes to the new `*-tools` packages added in this PR (`cuda-crt-tools` and `cuda-nvvm-tools`), they are used to construct `cuda-nvcc-tools`, which is still what `cuda-nvcc_{{ target_platform }}` uses in CUDA 12.2. So this should still generate a functional compiler. Though please feel free to recheck if I've missed something